test/verify/check-storage-swap

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import unittest
+import parent  # noqa: F401
+from storagelib import StorageCase
+from testlib import test_main, nondestructive
+from testvm import DEFAULT_IMAGE
+
+
+@nondestructive
+@unittest.skipIf(not DEFAULT_IMAGE.startswith("fedora"), "No zram by default")
+class TestStorageswap(StorageCase):
+
+    def testZram(self):
+        m = self.machine
+        b = self.browser
+
+        self.assertEqual(m.execute("lsblk -n -o MOUNTPOINTS /dev/zram0").strip(), "[SWAP]")
+
+        # Cockpit should recognize that zram0 is used as swap
+        # eventhough it does not have a swap header on it.
+        self.login_and_go("/storage")
```

Comment:
This should say "eventhough udev ignores it and does not set the ID_* attributes for it"